markdown/chapters/meandeck/brainstorm.md

```diff
@@ -119,12 +190,83 @@ It follows the exact sequence as the one before but you cycle into the
 Brainstorm rather into the mana needed for it. Once again, the same trick can be
 done if you have excess mana or the card X is a cantrip already in exchanging
 the cantrip for mana within the pile to enable the playing around of removal to
-ensure you have a 0 card library on resolution of the Oracle trigger.
+ensure you have perfect pile to generate a zero card library on resolution of
+the Oracle trigger.
+
+## Brainstorm Pile - Not using LED.
+
+Sometimes you may not wish to commit to a Brainstorm pile because you are
+concerned about interaction and are holding Force of Will and a *pitch* card for
+it. In these circumstances you can often create an imperfect Brainstorm pile to
+win that same turn whilst keeping up your countermagic. Let us consider the
+following scenario:
+
+You successfully resolve Doomsday but suspect your opponent may be holding an
+{{Endurance}} plus *pitch* card in hand. In this scenario you can build the
+following pile:
+
+> {B}{B}{B} + Brainstorm + Street Wraith + X + X
+
+<row variant="pile">{{!EoA}} {{!LP}} {{!LP}} {{!LP}} {{!TO}}</row>
+
+1. Cycle SW, draw the Edge of Autumn (EoA)
+1. Cycle EoA, draw the LP and cast it
+1. Cast Brainstorm off of LP and put back X then X
+1. Cast both LP, crack them for {U}{U}
+1. Cast TO with two cards left in your library
+
+Here we successfully perform an imperfect Brainstorm pile without needing to
+discard our hand to LED.
+
+You can also utilise a spare land in hand plus a land drop to enable the
+creation of a perfect Brainstorm pile. Consider the following scenario where we
+have perhaps run out of fetchable lands from our deck as they are in our hand:
+
+> {B}{B}{B} + Brainstorm + Street Wraith + Underground Sea + Underground Sea  
+> A fetchland in play, land drop available for turn
+
+<row variant="pile">{{!SW}} {{!LP}} {{!Misty Rainforest}} {{!Daze}} {{!TO}}</row>
+
+1. Cycle SW, draw SW
+1. Cycle SW, draw the LP and cast it
+1. Cast Brainstorm off of LP and put back both Underground Seas
+1. Play Misty Rainforest, crack both fetchlands
+1. Tap both Seas for {U}
+1. Cast TO with zero cards left in your library
+
+## Brainstorm Pile - Speeding up a PTT Pile
+
+Sometimes you might not be able to execute an imperfect Brainstorm pile due to
+known removal and so elect to build a pass the turn pile instead. You may have
+the Brainstorm in hand but elect to build a slower pile relying on cycle effects
+and fetch lands. If, for whatever reason, circumstances in the game state
+change, you can use the Brainstorm to dig into the pile quicker to access what
+you need to win on the spot.
+
+Let us consider the following example:
+
+> {B}{B}{B} + Brainstorm + X
+
+<row variant="pile">{{!SW}} {{!SW}} {{!CoS}} {{!TO}} {{!Pact of Negation}}</row>
```

Comment:
```suggestion
<row variant="pile">{{!SW}} {{!SW}} {{!CoS}} {{!TO}} {{!PoN}}</row>
```

I'll merge #113 first so that `PoN` is readily available 🤓

Reply:
Alrighty, changed